transport/tchannel/channel_inbound_test.go

```diff
@@ -65,7 +76,10 @@ func TestChannelInboundStartAlreadyListening(t *testing.T) {
 
 	i := x.NewInbound()
 
-	i.SetRouter(new(transporttest.MockRouter))
+	mctrl := gomock.NewController(t)
+	router := transporttest.NewMockRouter(mctrl)
+	router.EXPECT().Procedures().Return(nil)
```

Comment:
Instead of building a MockRouter that expects a read-only operation, can you
just use an empty MapRouter?